lib/assets/javascripts/cdb

```diff
@@ -1 +1 @@
-Subproject commit a2b8834622f2b4d3b849a7e810cff998d982844d
+Subproject commit 5bee7c8c6c6b2a62ec29c598f9c4331a9cb831fb
```

Comment:
I'm definitely rebased to the updated bbg_production branch. What do I have to do to pull in our custom cartodb.js, however?